src/components/common/DateDisplay/index.tsx

```diff
@@ -11,20 +14,27 @@ const IconWrapper = styled(FontAwesomeIcon)`
 interface DateDisplayProps {
   date: Date
   showIcon?: boolean
+  tooltipPlacement?: Placement
 }
 
-export function DateDisplay({ date, showIcon }: DateDisplayProps): JSX.Element {
+export function DateDisplay({ date, showIcon, tooltipPlacement = 'top' }: DateDisplayProps): JSX.Element {
+  const { tooltipProps, targetProps } = usePopperDefault<HTMLInputElement>(tooltipPlacement)
   // '5 days ago', '1h from now' date format
   const distance = formatDistanceToNowStrict(date, { addSuffix: true })
   // Long localized date and time '04/29/1453 12:00:00 AM'
   // P: Long localized date: 04/29/1453
   // pp: Long localized time: 12:00:00 AM
   // For reference: https://date-fns.org/v2.17.0/docs/format
-  const fullLocaleBased = format(date, 'P pp')
-
+  const fullLocaleBased = format(date, 'P pp zzzz')
+  const previewDate = format(date, 'd MMMM yyyy - h:mm a')
   return (
     <span className="wrap-datedisplay">
-      {showIcon && <IconWrapper icon={faClock} />} <span>{distance}</span> <span>({fullLocaleBased})</span>
+      <Tooltip {...tooltipProps}>
+        {distance} - {fullLocaleBased}
```

Comment:
Looks great! 

Maybe we can increase the tooltip width to prevent this:

![image](https://user-images.githubusercontent.com/622217/135081526-5fed1be1-f82c-470f-a337-36de17a8af74.png)

Or, add a <br> between the two parts.